lib/reporters/json.js

```diff
@@ -0,0 +1,16 @@
+/*
+ * Author: Josh Hagins
+ * https://github.com/jawshooah
+ */
+
+'use strict';
+
+module.exports  = {
+  reporter: function(results) {
```

Comment:
Can we simplify this and the other reporters to exports the function directly? I don't see any value in the additional object with its `reporter` property. Applies here, for the checktyle reporter and the custom one (in tests).